### ISSUE TITLE
data leakage

### ISSUE THREAD

**Issue by ChunyangCLi** [chl8856/Dynamic-DeepHit#5]
In the implementation and experiments, data normalization/standardization is done before splitting into train and test set, that could result in test set information leaked into training set...

**Comment by 2mLi** [comment]
Dear Chunyang, 

I have noticed the same problem and I just commented to remove the normalisation/standardisation part on my own fork. I think the code still achieved some good levels of performances (only in terms of c-index and Briers) without the author's version of normalisation, but that's after I applied my own versions of normalisations to my own data. I have not tested whether the performance of the code would differ on the PBC2 dataset. 

It did not seem that the author of the repo still logged on to their GitHubs any longer, but I also had a question regarding the code and it would be appreciated if you or anyone else could provide any comments.
